Exiled.Events/Commands/Reload/Translations.cs

```diff
@@ -0,0 +1,53 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="Translations.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Commands.Reload
+{
+    using System;
+
+    using CommandSystem;
+
+    using Exiled.Loader;
+    using Exiled.Permissions.Extensions;
+
+    /// <summary>
+    /// The reload configs command.
+    /// </summary>
+    public class Translations : ICommand
+    {
+        /// <summary>
+        /// Gets static instance of the <see cref="Translations"/> command.
+        /// </summary>
+        public static Translations Instance { get; } = new Translations();
+
+        /// <inheritdoc/>
+        public string Command { get; } = "translations";
+
+        /// <inheritdoc/>
+        public string[] Aliases { get; } = Array.Empty<string>();
+
+        /// <inheritdoc/>
+        public string Description { get; } = "Reload plugin translations.";
+
+        /// <inheritdoc/>
+        public bool Execute(ArraySegment<string> arguments, ICommandSender sender, out string response)
+        {
+            if (!sender.CheckPermission("ee.reloadtranslations"))
+            {
+                response = "You can't reload translations, you don't have \"ee.reloadtranslations\" permission.";
+                return false;
+            }
+
+            bool haveBeenReloaded = TranslationManager.Reload();
+
+            Handlers.Server.OnReloadedTranslations();
+
+            response = "Plugin translations have been reloaded successfully!";
```

Comment:
The response should be different when `haveBeenReloaded` is false.

Reply:
This is an issue with all of the reload commands and should be fixed in a separate PR.